aptos-move/block-executor/src/executor.rs

```diff
@@ -1394,24 +1399,21 @@ where
                     if let Some(commit_hook) = &self.transaction_commit_hook {
                         commit_hook.on_execution_aborted(idx as TxnIndex);
                     }
-                    // Record the status indicating abort.
-                    return Err(BlockExecutionError::FatalVMError((err, idx as TxnIndex)));
+                    alert!("Fatal VM error by transaction {}", idx as TxnIndex);
+                    // Record the status indicating the unrecoverable VM failure.
+                    return Err(BlockExecutionError::FatalVMError(err));
+                },
+                ExecutionStatus::DelayedFieldsCodeInvariantError(msg) => {
+                    return Err(BlockExecutionError::FallbackToSequential(
+                        PanicOr::CodeInvariantError(msg),
```

Comment:
we generally should call `code_invariant_error(msg)` instead of creating directly (if you need you can add .into(), if it doesn't get casted automatically from Panic to PanicOr)

Reply:
braindead editing, thanks for catching

Reply:
it's so dumb that we can't make types uninstantiable - only way is via visibility and it doesn't work if you need to include the type elsewhere.